Doc/library/functions.rst

```diff
@@ -628,7 +634,7 @@ are always available.  They are listed here in alphabetical order.
    elements of *iterable* for which *function* returns false.
 
 
-.. class:: float([x])
+.. class:: float(x=0.0)
```

Comment:
```suggestion
.. class:: float(x=0.0, /)
```

```
>>> float(x=3)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: float() takes no keyword arguments
```

Reply:
I think the `,/ ,` and the `, *,` could be specified for the other functions where they make sense, I'll try it. I fear it make it harder to read and we may loose the initial "readability" effect. But in the other hand it'll teach something usefull to the reader.